### PR TITLE
Update solvers.md

### PR DESCRIPTION
**Pull request by gogonimago**
# Description

# Changes

- [ ] ...
- [ ] ...

### CODE REVIEW

File: docs/cow-protocol/concepts/introduction/solvers.md [hunk 1]
```diff
@@ -10,16 +10,9 @@ CoW Protocol delivers optimal price outcomes by leveraging an open solver compet
...
 
-Solvers are compensated in COW tokens for settling batches, incentivizing them to compete to find better prices and win the right to execute user intents. 
-
-## How Solvers Work
-
-Solvers can move tokens on behalf of the user using the ERC-20 approvals that the user granted to the [vault relayer](/cow-protocol/reference/contracts/core/vault-relayer) contract.
-The [settlement](/cow-protocol/reference/contracts/core/settlement) contract, meanwhile, verifies the signature of the user's intent and ensures that execution happens according to the limit price and quantity specified by the user.
-
-Anyone with some DeFi knowledge and ability to code an optimizations algorithm can create a solver.
+In order to incentivize a healthy competition and guarantee the best outcome for user orders, solvers get awarded COW tokens each time they solve a batch.
```
Comment (mfw78) [inline docs/cow-protocol/concepts/introduction/solvers.md]:
```suggestion
In order to incentivize a healthy competition and guarantee the best outcome for user orders, solvers get rewarded COW tokens each time they solve a batch. 
```

**Review by mfw78 — COMMENTED** [review]
Minor nit

**Review by mfw78 — APPROVED** [review]

**Review by mfw78 — APPROVED** [review]

**Comment by gogonimago** [comment]
Do I need to do anything more on this PR? I approved your nit

**Review by mfw78 — APPROVED** [review]

**Comment by mfw78** [comment]
> Do I need to do anything more on this PR? I approved your nit

There was an edit that was also done by @marshymarsh that resulted in a conflict (the edit had been merged some time ago). I resolved the conflict and merged all.

**Pull request merged**